agent/engine/docker_task_engine.go

```diff
@@ -326,6 +337,14 @@ func (engine *DockerTaskEngine) sweepTask(task *api.Task) {
 			seelog.Errorf("Error removing container reference from image state: %v", err)
 		}
 	}
+
+	// Clean metadata directory for task
+	if engine.cfg.ContainerMetadataEnabled {
+		err := engine.metadataManager.Clean(task.Arn)
+		if err != nil {
+			seelog.Errorf("Clean task metadata failed for task %s: %v", task, err)
```

Comment:
Can you please change this to a `Warn`? I'd prefer `Error` to be used if it correlates with a customer facing error, which this is not (Applies to the rest of the file(s) as well).

Reply:
fixed.